OpenGL example in README doesn't work

This example appear in the README:

```
let sdl_context = sdl2::init().unwrap();
let video_subsystem = sdl_context.video().unwrap();
let window = video_subsystem.window("Window", 800, 600)
    .opengl()
    .build()
    .unwrap();
let canvas = window.into_canvas()
    .index(find_sdl_gl_driver().unwrap())
    .build();

gl::load_with(|name| video_subsystem.gl_get_proc_address(name) as *const _);
canvas.window().gl_set_context_to_current();
```

But I get an error like this:

```
experiments)
error[E0599]: no method named `window` found for type `std::result::Result<sdl2::render::Canvas<sdl2::video::Window>, sdl2::IntegerOrSdlError>` in the current scope
  --> src/main.rs:32:12
   |
32 |     canvas.window().gl_set_context_to_current();
   |            ^^^^^^

error: aborting due to previous error
```

It looks like the code needs another `unwrap()`.

--- a/README.md
+++ b/README.md
@@ -354,7 +354,8 @@ let window = video_subsystem.window("Window", 800, 600)
     .unwrap();
 let canvas = window.into_canvas()
     .index(find_sdl_gl_driver().unwrap())
-    .build();
+    .build()
+    .unwrap();
 
 gl::load_with(|name| video_subsystem.gl_get_proc_address(name) as *const _);
 canvas.window().gl_set_context_to_current();

